modules/core/src/main/java/org/apache/ignite/internal/cluster/IgniteClusterImpl.java

```diff
@@ -313,7 +315,7 @@ public IgniteClusterImpl(GridKernalContext ctx) {
         guard();
 
         try {
-            ctx.state().changeGlobalState(active, serverNodes(), false).get();
+            ctx.state().changeGlobalState(active ? ACTIVE : INACTIVE, true, serverNodes(), false, false).get();
```

Comment:
Why `forceDeactivation` should be `true` for cluster activate?

Reply:
All previous internal invocations were considered forced. I think we should not change it where not required. Not-forced call are considered only for user commands through command.sh/Rest .